src/main/webapp/scripts/forum-template.js

```diff
@@ -0,0 +1,29 @@
+const forumTemplate = '<div class="forum-element {{elementType}}">\n' +
```

Comment:
Is it possible to put the template in an HTML file? That would make formatting and reviewing much easier.

Reply:
We originally had it in the html file in a script tag but the linter gave us errors.  Should I move it back and proceed with errors?

Reply:
It's fine, Niha told me about the linter issue. Can you add a comment in this file with a note on that? I.e. why it's in JS but not in HTML.

Reply:
Added comment to javascript file.